### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by tkachuktaras**

### CODE REVIEW

File: article_1.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                    <div class="upper">
+                        <img src="../images/user.png" class="user__img">
+                        <textarea id="textarea__comment" placeholder="Оставьте комментарий"></textarea>
+                    </div>
+                    <div class="under">
+                        <input type="submit" class="comment__button">
+                    </div>
+                </div>
+
+                <div class="written__comments">
```
Comment (yurykozhenov) [inline article_1.php]: Use for loop and list to generate comments

File: blocks/header.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<body>
```
Comment (yurykozhenov) [inline blocks/header.php]: `<body>` is not needed here

File: css/article.css [hunk 3]
```diff
@@ -15,14 +17,19 @@
...
 
+
+/* Center Part*/
+
 .center__part {
     margin-top: -8px;
     padding-top: 15px;
     padding-left: 20px;
     padding-right: 20px;
+    padding-bottom: 10px;
```
Comment (yurykozhenov) [inline css/article.css]: Instead of 4 different paddings, use single `padding` property

File: login.php [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+<head>
+    <title>Авторизация</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css?423">
+    <link rel="stylesheet" href="css/login_page.css?423">
+    
+    <link rel="icon" href="./images/ico.png" type="image/png">
+
+    <script src="https://kit.fontawesome.com/46e75e08cb.js" crossorigin="anonymous"></script>
```
Comment (yurykozhenov) [inline login.php]: You can add another php file with some common `<head>` stuff

File: article_1.php (hunk 1 above)
Comment (tkachuktaras) [reply article_1.php]: yes sir

**Pull request merged**